usps_service/__main__.py

```diff
@@ -0,0 +1,146 @@
+                                               city=address[args.city],
+                                               state=address[args.state],
+                                               zip5=address[args.zip5],
+                                               zip4=zip4)
+
+        if result_code == 200 or result_code == 201:
+            result_fields = ['street', 'suite', 'city',
+                             'state', 'zip5', 'zip4']
+
+            for field in result_fields:
+                address["result " + field] = result[field]
+        else:
+            address["error"] = result[error]
+
+        sys.stdout.write(f'''\r{str(i+1)} of {str(len(address_list))} complete''')
+
+    sys.stdout.write("\n")
+    return address_list
+
+def load_addresses(args):
+    """
+    Open the provided csv file and load the data from it into a list of
+    OrderedDicts.
+
+    :param dict args: the arguments provided by the user. Must contain an
+    args.file str with a file path name
+    :return: a list of input from the csv, each line in its own OrderedDict.
+    The list may contain extraneous data besides the address itself.
+    Each address within the list is an OrderedDict where keys are the headers
+    on the first line of the csv file.
+    """
+    address_list = []
+    with open(args.file, 'r') as file:
+        reader = csv.DictReader(file, delimiter=',')
+        for row in reader:
+            address_list.append(row)
+
+    return address_list
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description='call on the usps-service to assist with address handling'
+        )
+    parser.add_argument('id',
+        help='a USPS issued ID code')
+
+    parser.add_argument('--file', '-f', '-F',
+        help='A file of addresses to standardize as a batch',
+        dest='file',
+        required=True)
+    parser.add_argument('--streetColumn', '--street',
+        help='The header title of the column holding the street address',
+        dest='street',
+        required=True)
+    parser.add_argument('--suiteColumn', '--suite',
+        help='The header title of the column holding the suite, box, etc.',
+        dest='suite',
+        required=False)
+    parser.add_argument('--cityColumn', '--city',
+        help='The header title of the column holding the city name',
+        dest='city',
+        required=True)
+    parser.add_argument('--stateColumn', '--state',
+        help='The header title of the column holding the state abreviation',
+        dest='state',
+        required=True)
+    parser.add_argument('--zip5Column', '--zip5',
+        help='The header title of the column holding the zip5',
+        dest='zip5',
+        required=True)
+    parser.add_argument('--zip4Column', '--zip4',
+        help='The header title of the column holding the zip4',
+        dest='zip4',
+        required=False)
+
+    args = parser.parse_args()
+    
+    standardize_batch(args)
```

Comment:
Could you add linting to the testing process? It will catch this error (expected newline at end of file).